RealmSwift/Migration.swift

```diff
@@ -80,7 +80,7 @@ extension Realm {
  instance provides access to the old and new database schemas, the objects in the Realm, and provides functionality for
  modifying the Realm during the migration.
  */
-public final class Migration {
+public struct Migration {
```

Comment:
I don't see any way for it to break anything, since they're immutable types which the user can't construct directly. `Realm` should maybe also be a struct but that would actually be a breaking change.